app/controllers/contributions_controller.rb

```diff
@@ -0,0 +1,17 @@
+class ContributionsController < ApplicationController
+  skip_before_action :authenticate_user!, only: %i[index]
+
+  def index
+    speaker_ids_with_pending_github_suggestions = Suggestion.where("json_extract(content, '$.github') IS NOT NULL").where(suggestable_type: "Speaker").pluck(:suggestable_id)
+
+    speakers = Speaker.where(github: "").order(talks_count: :desc)
+    @speakers_without_github_count = speakers.count
+    @speakers_without_github = speakers.where.not(id: speaker_ids_with_pending_github_suggestions).limit(11)
```

Comment:
I think we should try to randomize a bit the speakers we display here so that we don't end up stuck with only very difficult speakers to identify